wandb/sdk/wandb_run.py

```diff
@@ -182,6 +183,23 @@ def join(self) -> None:
         self._retry_thread.join()
 
 
+# TODO(kpt) should we auto decorate?
```

Comment:
i'm leaning to not add it... but not sure yet

Reply:
sg, keep it handy just in case you change your mind :)